crates/solidity/outputs/cargo/crate/build.rs

```diff
@@ -2,10 +2,10 @@
 //! It is removed when publishing to crates.io.
 
 use anyhow::Result;
-use codegen_grammar::Grammar;
+use codegen_grammar::{Grammar, GrammarConstructorDslV2};
 use codegen_parser_generator::{AstModel, RustGenerator};
 use infra_utils::cargo::CargoWorkspace;
-use solidity_language::{GrammarConstructorDslV2, SolidityDefinition};
+use solidity_language::SolidityDefinition;
 
 fn main() -> Result<()> {
```

Comment:
nit/not blocking:

Since we are doing that refactoring, what do you think about changing `RustGenerator` itself to take a `language` (v2) instance instead, and lowering it to the legacy `grammar`?
This way, output crates don't need to know about this, and won't be affected when the codegen/backend is eventually fixed, and we delete `codegen_grammar`.